.github/workflows/nl-action.yml

```diff
@@ -24,6 +24,11 @@ jobs:
       with:
         path: ${{ env.pythonLocation }}
         key: ${{ env.pythonLocation }}-${{ hashFiles('setup.py') }}-${{ hashFiles('requirements.txt') }}
+    - name: Install Tesseract
+      run: |
+        sudo apt-get update
+        sudo apt-get install tesseract-ocr libtesseract-dev libleptonica-dev pkg-config \
+          tesseract-ocr-eng tesseract-ocr-osd tesseract-ocr-deu tesseract-ocr-fra tesseract-ocr-spa
```

Comment:
@mnamysl : It's not ideal to do the tesseract installation in the workflow. We're installing only light transformations/filters by default. If this requirement can be specified in the requirements.txt, it should be automatically picked up. Let me find a better way to do this.

Reply:
@AbinayaM02 : Thank you for your feedback.

How about a mechanism similar to that used for the pip packages, e.g.:
`if [ -f packages.txt ]; then sudo apt-get update; xargs sudo apt-get install -y < packages.txt; fi` ?

Reply:
The idea is whatever is supposed to be installed for your transformation should go only in your requirements.txt. Let me pull your code and see if this can be done at transformation level through requirements.txt. You can also try to find a way. We'll update once we find something.

Reply:
@mnamysl: Since tesseract can be installed only through apt-get install, it makes sense to do what you're already doing for installation. But, would it be too much if I request you to explore pytesseract (python wrapper of tesseract)? If pytesseract works as expected, then we can switch to that without making changes to the workflow. Let me know what you think.

Reply:
@AbinayaM02 : Thank you for your feedback!

Unfortunatelly, [pytesseract](https://pypi.org/project/pytesseract/) also [requires](https://github.com/madmaze/pytesseract#installation) Tesseract to be installed in the OS.

I've also tested one open-source neural OCR model but it performed much worse, thus I stick to Tesseract.
Finding a comparable accurate neural model would be a nice extension but I would leave it for future work (another PR).

Generally, it is not bad to have a method that allows us to install system packages (in addition to purely python ones).
Other transformation might use, e.g., another tool or library like spell-checker engine, which is not available in the default OS image.

Reply:
Oh yeah! Forgot that pytesseract will require Tesseract installation. 

By current design, installing NL-Augmenter will by default install only all the light transformations and filters. The heavy transformations and filters should be installed based on users need. If we make your transformation heavy, we can provide the instructions to install tesseract in the README. What do you suggest? Should we make your transformation as heavy since it uses Tesseract?

Reply:
@AbinayaM02: Thank you for your input - I definitely agree that it would be the best option.

I reverted the changes in the workflow and flagged _ocr_perturbation_ as heavy in `test/mapper.py`.

I also updated README.md file with the instructions to install Tesseract on Ubuntu and included a link to the instructions for other OS.

Reply:
Great! Thank you for making those changes.